### ISSUE TITLE
Build failure in certain Windows environment

### ISSUE THREAD

**Issue by jgm** [jgm/cmark#145]
I'm seeing a build failure in the Windows environment used by Haskell's stack build tool for the Haskell cmark bindings.  Build log here:
https://ci.appveyor.com/project/jgm/cmark-hs/build/1.0.4

@vmg this was introduced by your commit ab6c81b960e86b26c7fda366f51ff29d1683a555

@nwellnhof any ideas?  I don't know much about Windows development.

EDIT:  This is not a cmake-based build, because we don't want to require cmake for creation of the Haskell bindings.  The sources are included in the package and a default build procedure is used.  This worked fine with previous versions.

**Comment by nwellnhof** [comment]
It seems that `ssize_t` isn't used anyway, so the following code can be simply removed from `config.h.in`:

~~~ c
#ifdef _WIN32
# include <BaseTsd.h>
  typedef SSIZE_T ssize_t;
  typedef SIZE_T size_t;
#else
# include <sys/types.h>
#endif
~~~

**Comment by jgm** [comment]
But that defines `size_t` also, and `size_t` is used...

**Comment by nwellnhof** [comment]
`size_t` is part of C89. There's no need for a typedef.

**Comment by jgm** [comment]
OK, thanks, let me try it.  I wonder why @vmg included the `size_t` typedef for WIN32 if it isn't needed?

**Comment by nwellnhof** [comment]
I think in some earlier version of @vmg's pull request, `ssize_t` was used for `bufsize_t`. The typedef for `size_t` was probably just an oversight.

**Comment by jgm** [comment]
Thanks.  Your fix works.

**Comment by jgm** [comment]
Released 0.26.1.
